### ISSUE TITLE
Microsector analysis

### ISSUE THREAD

**Issue by TerminatorPlati** [theOehrly/Fast-F1#23]
Hi, with Fast-F1 is it possible to create an analysis of the Microsectors? Let me explain better, a map of the circuit where the driver who made the best micro-sector is highlighted, for example in Q3. I don't know if I've made myself clear

**Comment by ricky732** [comment]
@TerminatorPlati did you finish the smooth you were working to? I did one too, but I am not so happy, is yours good?

**Comment by theOehrly** [comment]
There is no ready-made micro sector analysis available with fastf1 currently. The api doesn't provide proper timing data for micro sectors.
 
In theory, you can define your own micro sectors and calculate the timing data from the car's speed. But in reality this is very difficult because of the low sample rate and because small accuracy problems tend to add up.
I have played around with creating a continuous speed delta map. This too allows you to see which car is faster where but doesn't require as much accuracy from the raw data. But still, I could basically produce any output I wanted if I wasn't careful.
In short, the accuracy of the raw data is the problem. Even if it can be done, it is very difficult to judge whether a result can be trusted. The output may look good and still be completely wrong.

**Comment by hash55** [comment]
> @TerminatorPlati did you finish the smooth you were working to? I did one too, but I am not so happy, is yours good?

Hi! Sorry to re-open this issue. Did you manage to do something accurate with microsectors? When I want to compare the delta time of two laps currently, I use `fastf1.utils.delta_time`,. However, I don't think it's accurate at all. Please let me know if you guys find something better
